package.json

```diff
@@ -17,7 +17,8 @@
     "build": "node tasks/build/runBuild",
     "build:gaia": "cd tasks/build/Gaia; ./runBuild.sh",
     "local": "yarn run testnet local",
-    "testnet": "node tasks/testnet.js",
+    "start": "node tasks/testnet.js",
+    "testnet": "yarn start",
```

Comment:
Nice! :) Maybe add `echo "'yarn testnet' will be deprecated. Please use 'yarn start'"`?

Reply:
gets lost if `yarn testnet` falls back to `yarn start`, think maybe  it's better to just print the message